src/schema/project-checkbook.ts

```diff
@@ -0,0 +1,21 @@
+import { char, numeric, pgTable, text, uuid } from "drizzle-orm/pg-core";
+import { z } from "zod";
+import { managingCodeEntitySchema } from "./managing-code";
+
+export const projectCategory = pgTable("project_category", {
+  id: uuid("id"),
+  managingCode: char("managing_code", { length: 3 }),
+  projectId: text("project_id"),
+  value: numeric("value"),
+});
+
+export const projectCategoryEntitySchema = z.object({
+  id: z.string().uuid(),
+  managingCode: managingCodeEntitySchema,
+  projectId: z.string(),
+  value: z.string().regex(RegExp("^[0-9]")),
```

Comment:
In this case, the values truly are numbers and we can use the ['number' validator](https://zod.dev/?id=numbers)